utils/python/CIME/test_status.py

```diff
@@ -241,7 +242,10 @@ def get_overall_test_status(self, wait_for_run=False, check_throughput=False, ch
                         rv = NAMELIST_FAIL_STATUS
 
                 elif (rv in [NAMELIST_FAIL_STATUS, TEST_PASS_STATUS] and phase.startswith(COMPARE_PHASE)):
-                    rv = TEST_DIFF_STATUS
+                    if (phase.startswith(COMPARE_BASELINE_PHASE) and TEST_BFAIL_STATUS in comment):
```

Comment:
Good point @jgfouca . (It's possible that I was the one who suggested something like this.) So I think your point is that this should just be changed back to giving a DIFF overall status if the baselines do not exist? I'd support that. (But then again, I personally care more about the status of the individual pieces than the overall status in my parsing, so maybe someone else like @ekluzek - who I think relies more on the overall status - should weigh in on this.)